src/js/components/Chart/__tests__/Chart-test.js

```diff
@@ -171,8 +188,84 @@ describe('Chart', () => {
     expect(tree).toMatchSnapshot();
   });
 
-  test('calcs', () => {
-    const result = calcs([1, 2, 3]);
+  test('calcs basic', () => {
+    const result = calcs([
+      [1, 2, 2],
+      [2, 2, 2],
+    ]);
+    expect(result).toMatchSnapshot();
+  });
+
+  test('calcs with single value', () => {
+    const result = calcs([1]);
+    expect(result).toMatchSnapshot();
+  });
+
+  test('calcs with negative min', () => {
+    const result = calcs([
+      [1, -2, -2],
+      [2, 2, 2],
+    ]);
+    expect(result).toMatchSnapshot();
+  });
+
+  test('calcs large thickness', () => {
+    const vals = [];
+    for (let i = 0; i < 8; i += 1) {
+      vals[i] = [];
+      vals[i].push(1);
+      vals[i].push(2);
+      vals[i].push(3);
+    }
```

Comment:
Not sure why vals[i] is being pushed to 3 times. Also you do something like this instead so you don't have to loop:
> let vals = Array(8).fill(1);

Reply:
`vals[i]` is being pushed three times because `vals` needs to be a 2D array with each element being an array of 3 numbers, so we need to push three numbers into each index. Also I tried to use `Array.fill` but I wasn't able to get it to work with filling 2D arrays

Reply:
You could probably do
let vals = Array(8).fill([1, 2, 3])

Reply:
@MikeKingdom Thanks that worked!